config/sync/captcha.settings.yml

```diff
@@ -1,12 +1,10 @@
 _core:
   default_config_hash: pfrv6lPS72f5WG1UB4wHkgQV2invlHwVpaCy_mY-ClQ
-enabled_default: 0
 default_challenge: captcha/Math
 description: 'This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.'
 administration_mode: false
 allow_on_admin_pages: false
 add_captcha_description: true
-wrong_captcha_response_message: 'The answer you entered for the CAPTCHA was not correct.'
```

Comment:
Not sure why this file was touched. Any ideas?

Reply:
It was added when bouncing between branches. Needs to be tossed out.